libHttpClient.props

```diff
@@ -36,7 +36,7 @@
     <HCLibToolset Condition="'$(PlatformToolset)'=='v140'">140</HCLibToolset>
     <HCLibToolset Condition="'$(PlatformToolset)'=='v141'">141</HCLibToolset>
     <HCLibToolset Condition="'$(PlatformToolset)'=='v142'">142</HCLibToolset>
-    <HCLibToolset Condition="'$(PlatformToolset)'=='v143' AND '$(HCLibPlatformType)' == 'GDK'">142</HCLibToolset>
+    <HCLibToolset Condition="'$(PlatformToolset)'=='v143'">143</HCLibToolset>
 
```

Comment:
This props file is only used when building from source, correct?  We can still ensure the GDK props file links properly with the v142 version

Reply:
might be true but check the gdk props file.  i forget if it uses this props in its path.  it might but forget.

Reply:
It does not use this props file.  It directly lists libHttpClient as a dependency.  It does depend on $(ExtensionPlatformToolset) so that will have to correctly handle v143, but that is outside the scope of this libHttpClient change.

Reply:
Just confirmed that $(ExtensionPlatformToolset) will be correctly set to 142, forcing titles to link against libHttpClient.142.* even when they are targeting VS toolset 143.